src/Assets/Icons/Location.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from "react"
+
+export class Location extends Component<null, null> {
```

Comment:
Copied the pattern on Component per Icon from other Icons in here.

Reply:
FYI This is a prime example of where you could use a ‘stateless functional component’, rather than creating a class.